src/unix/linux_like/linux/gnu/mod.rs

```diff
@@ -433,6 +434,15 @@ s! {
         pub options: ::__u32,
     }
 
+    pub struct xsk_tx_metadata_completion {
```

Comment:
Everything added here is uapi right? It should probably go in `src/unix/linux_like/linux/mod.rs` rather than duplicating into gnu and musl. (The existing xdp types could probably move as well? Not in this PR though.)

Reply:
> Everything added here is uapi right?

Yes.

> The existing xdp types could probably move as well?

I think so, I wasn't sure where to put them when I initially added them. I'll create a PR after this one is merged to move the rest too.